kv/txn_coord_sender_test.go

```diff
@@ -35,18 +36,50 @@ import (
 	gogoproto "github.com/gogo/protobuf/proto"
 )
 
+// retryableLocalSender provides a retry option in the event of
+// range splits.
+type retryableLocalSender struct {
+	*LocalSender
+}
+
+// Send implements the client.Sender interface.
+func (rls *retryableLocalSender) Send(call *client.Call) {
+	// Instant retry with max two attempts to handle the case of a
+	// range split, which is exposed here as a RangeKeyMismatchError.
+	// If we fail with two in a row, pass the error up to caller and
+	// let the remote client requery range metadata.
```

Comment:
I cleaned up the comment and rearranged things so that we now create a retryableLocalSender via a `newRetryableLocalSender(*LocalSender, *testing.T)` method which should eliminate confusion about this being used anywhere outside unittests. The testing object is used to `Fatalf()` in the event of more than two attempts to send.